drivers/eeprom/eeprom_simulator.c

```diff
@@ -88,7 +90,7 @@ static int eeprom_fd = -1;
 static const char *eeprom_file_path;
 static const char default_eeprom_file_path[] = "eeprom.bin";
 #else
-static u8_t mock_eeprom[DT_INST_0_ZEPHYR_SIM_EEPROM_SIZE];
+static u8_t mock_eeprom[DT_INST_REG_SIZE(0)];
```

Comment:
Shouldn't this be `DT_INST_PROP(0, size)`?

Reply:
Yep, fixed.